lib/auth/AuthBasic.cc

```diff
@@ -41,6 +41,14 @@ std::string base64_encode(const std::string& s) {
 AuthDataBasic::AuthDataBasic(const std::string& username, const std::string& password) {
     commandAuthToken_ = username + ":" + password;
     httpAuthToken_ = base64_encode(commandAuthToken_);
+    methodName_ = DEFAULT_BASIC_METHOD_NAME;
+}
```

Comment:
```suggestion
AuthDataBasic::AuthDataBasic(const std::string& username, const std::string& password)
    : AuthDataBasic(username, password, DEFAULT_BASIC_METHOD_NAME) {}
```

Reuse the other constructor.